### PR TITLE
New functionality to write file into a file so that external browser …

### PR DESCRIPTION
**Pull request by abcdec**
…can load output.

Example setting Native Emacs + Firefox (with Auto Reload plug-in):

```
            (setq plantuml-external-browser-path "C:/Program Files (x86)/Mozilla Firefox/firefox.exe")
            (setq plantuml-file-output-type 'png)
            (setq plantuml-file-output-path "C:/Windows/Temp/plantuml-mode-output.png")
            (setq plantuml-external-browser-args (list (concat "file:///" plantuml-file-output-path)))
```

image

C-cC-c will invoke plantuml as usual but instead of normal preview inside emacs, it will write output into a file specified by plantuml-file-output-path with type specified by plantuml-file-output-type.

M-x plantuml-start-external-browser

invokes preferred browser to view.  Use of browser to do automatic refresh will allow user to see update semi-real time. (e.g. Firefaxo Auto Reload plug-in.)

(If alternative implementation / design is preferred, reject pull and send it to issue list for further discussion.)

This is re-submission of pull request #37

Example variable setting for cygwin bash Emacs + + Firefox (with Auto Reload plug-in).

```
            (setq plantuml-file-output-path "/tmp/plantuml-mode-output.png")
            (setq plantuml-external-browser-path "/cygdrive/c/Program Files (x86)/Mozilla Firefox/firefox.exe")
            (setq plantuml-external-browser-args (list (concat "file:///C:/cygwin64" plantuml-file-output-path)))
            (setq plantuml-file-output-type 'png)
```

image

### CODE REVIEW

**Comment by skuro** [comment]
The idea is interesting, and I can see the benefits of the workflow you describe. Still, I would not do it just as you suggest here.

**tl;dr:** saving to a file is good, but it has to be a buffer-local configuration. Managing the browser is not good.

I'll close this one for now, we can continue the discussion on #46.
